File: AOJ/ALDS1/1/c_prime_number.cpp
#include <bits/stdc++.h>
using namespace std;

bool is_prime(int n)
{
    for (int i = 2; i <= sqrt(n); i++)
    {
        if (n % i == 0)
            return false;
    }
    return true;
}

int main()
{
    int n, a, count = 0;
    cin >> n;

    for (int i = 0; i < n; i++)
    {
        cin >> a;
        if (is_prime(a))
        {
            count++;
        }
    }
    cout << count << endl;
}

File: AOJ/ALDS1/1/c_prime_number.py
import math


def is_prime(a):
    sqrt_a = int(math.sqrt(a))
    for i in range(2, sqrt_a + 1):
        if a % i == 0:
            return False
    return True


def main():
    n = int(input())
    count = 0

    for i in range(n):
        a = int(input())
        if is_prime(a):
            count += 1

    print(count)


if __name__ == "__main__":
    main()
